# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: ipc/glue/ProtocolUtils.h
# Commit: f32859378512
# Full Hash: f32859378512460cb549cefd4aefaf4481467b5f
# Author: Markus Stange <mstange.moz@gmail.com>
# Date: 2020-11-10 09:57:49
# Description:
#   Bug 1675828 - Call OtherPidMaybeInvalid for the IPC marker because this code can run at times when the other pid is not known. r=jld
#   
#   This avoids a crash from the MOZ_RELEASE_ASSERT in OtherPid() on Android.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96262
# ==============================================================================

diff -r 20964901b728 -r f32859378512 ipc/glue/ProtocolUtils.h
--- a/ipc/glue/ProtocolUtils.h	Tue Nov 10 01:28:24 2020 +0000
+++ b/ipc/glue/ProtocolUtils.h	Tue Nov 10 01:45:51 2020 +0000
@@ -513,9 +513,9 @@
   already_AddRefed<nsISerialEventTarget> GetMessageEventTarget(
       const Message& aMsg);
 
- private:
   base::ProcessId OtherPidMaybeInvalid() const { return mOtherPid; }
 
+ private:
   int32_t NextId();
 
   template <class T>
